bitnami/spark/templates/NOTES.txt

```diff
@@ -0,0 +1,41 @@
+
+** Please be patient while the chart is being deployed **
+
+1. Get the Spark master WebUI URL by running these commands:
+
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "spark.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "spark.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "spark.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "spark.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:{{ .Values.master.webPort }} to use your application"
+  kubectl port-forward $POD_NAME {{ .Values.master.webPort }}:{{ .Values.master.webPort }}
+{{- end }}
+
+2. Submit an application to the cluster:
+
+To submit an application to the cluster the spark-submit script must be used. That script can be
+obtained at https://github.com/apache/spark/tree/master/bin.
+With the script run the following to submit the application:
+
+$ spark-submit --master spark://<Service-IP>:{{ .Values.master.clusterPort }} --deploy-mode cluster  /path/to/application 1000
```

Comment:
Instead of using Service-IP, you could also add instructions on how to get the service IP or nodePort. (You could re-use part of the instructions above)

Reply:
You are right that's not cool, I have changed it for a command that obtains the service IP.